Makefile

```diff
@@ -918,7 +922,7 @@ generate-gitignore: ## update gitignore files
 
 .PHONY: generate-images
 generate-images: | node_modules
-	npm install --no-save fabric@6 imagemin-zopfli@7
+	pnpm install --no-save fabric@6 imagemin-zopfli@7
```

Comment:
`pnpm install` does not have a `--no-save` option:

https://pnpm.io/cli/install

Reply:
Apparently there is open issue https://github.com/pnpm/pnpm/issues/1237 about it. I'm not sure how to solve.

Reply:
I guess what we should do is make a second `package.json` and lockfile inside the `tools` directory that holds these dependencies. That way we avoid any `pnpm install` failures related to those native modules on the main package.

Reply:
Good catch. I separated it out, and added cairo and pixman to the flake so that at least those who use nix can ensure they have the expected build deps. I'll have to add a mention of those two to the docs as turns out that prebuilt versions of canvas aren't available for at least my combination of devices and node versions.

Reply:
Looks good. I think there might be some wasm-based svg-to-png solution somewhere which would work everywhere, but this is good enough for now.